### DISCUSSION TITLE
Avatar padding, border?

### DISCUSSION THREAD

**Discussion by conx13** [quasarframework/quasar#16706] [General - Components / Directives / etc] [answered]
Newbie question: how to create a simple line around an image? The sample I found doesn't always work. In some places it's ok, but in some not... Would like to avoid styles.
`  <q-item-section avatar>
    <q-avatar color="grey" class="q-pa-xs">
      <q-img loading="eager" src="https://cdn.quasar.dev/img/avatar6.jpg" />
    </q-avatar>
  </q-item-section>`
[https://codepen.io/Hillar-Aas/pen/mdvZpQP?editors=101](url)

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/dyaBdoG?editors=1000

But why avoid styles? If you mean inline styles it might be a good idea, but is you mean avoid writing CSS then it's a bad idea.
